docs/onboarding/llm-analytics/_snippets/verify-llm-events-step.mdx

```diff
@@ -0,0 +1,19 @@
+<Step checkpoint title="Verify traces and generations" subtitle="Confirm LLM events are being sent to PostHog" docsOnly>
```

Comment:
We'd have to implement something to purposefully exclude these steps because they don't apply if you're already in the app. Hence the docsOnly